eZ/Publish/Core/REST/Server/Controller/Content.php

```diff
@@ -473,8 +470,6 @@ public function updateVersion($contentId, $versionNumber, Request $request)
 
         try {
             $this->repository->getContentService()->updateContent($versionInfo, $contentUpdateStruct);
-        } catch (ContentValidationException $e) {
-            throw new BadRequestException($e->getMessage());
         } catch (ContentFieldValidationException $e) {
             throw new BadRequestException($e->getMessage());
```

Comment:
@andrerom @dpobel Yep, work in progress. See updated description for current output.